src/current/_includes/releases/v22.2/v22.2.13.md

```diff
@@ -0,0 +1,138 @@
+## v22.2.13
+
+Release Date: August 9, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-13-security-updates">Security updates</h3>
+
+- The full set of TLS ciphers that was present in v22.1 have been included in the existing cipher suites list, which can be enabled with the `COCKROACH_TLS_ENABLE_OLD_CIPHER_SUITES` environment variable. [#105355][#105355]
+
+<h3 id="v22-2-13-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Fixed an issue where [changefeeds](../v22.2/change-data-capture-overview.html) emitting to a cloud storage sink with the [`compression` option](../v22.2/create-changefeed.html#compression-opt) may experience resource leakage (memory and go routines) when encountering transient errors. [#106856][#106856]
```

Comment:
Should these be moved to Bug Fixes?

Reply:
I never know if they should or not, but I have moved to bug fixes, since really they could be either and bug fix is probably more useful to the user... 👍